packages/scrubber-bar/karma.bs.config.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Question: can we use Jest as our test runner?

Reply:
So my first go at test configuration used jest, but I kept running into issues with it. I was having trouble importing ES Modules for testing and also with testing web components. The open-wc group suggests using karma and their template worked out of the box so I left it.

I'm happy to switch to jest if we can figure out how to make it test properly, I just spent a lot of time already :(